static/src/stylesheets/module/_adslot.scss

```diff
@@ -418,7 +418,7 @@
 }
 
 .ad-slot--carrot {
-    min-height: 250px;
+    min-height: 10px;
```

Comment:
Minor: I would set this to 0 since we want to override the existing min-height. Setting it to 10px will potentially puzzle someone in the future when they wonder `why 10px?`